<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesListEditor</class>
 <widget class="EntityWidget" name="NotesListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>324</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">EntityWidget {
    border: 1px solid #191919;
    border-left: 1px solid #455a64;
    border-top: 2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(69, 90, 100, 255), stop:0.498 rgba(69, 90, 100, 255), stop:0.502 rgba(24, 24, 24, 255), stop:1 rgba(24, 24, 24, 255));

    background-color: #37474f;

}

QTextBrowser {
    font: 11px &quot;Open Sans&quot;;
   color: #191919;
}</string>
  </property>
  <property name="respondable">
   <bool>true</bool>
  </property>
  <property name="parentPainting">
   <enum>E::PaintParentAfter</enum>
  </property>
  <layout class="QVBoxLayout" name="main_layout" stretch="0,1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>17</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="TextBrowser" name="textBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>272</width>
       <height>44</height>
      </size>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
    border-radius: 1px;
    background-color: #fafafa;
    padding: 2px;
border-radius: 2px;
}

QTextBrowser:!focus {
    border-radius: 2px;
    background-color: rgba(255, 255, 255, 204);
}

QTextBrowser:focus {
    border-radius: 2px;
    background-color: #ffffff;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="documentTitle">
      <string>@title_notes: %0</string>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="MarkersListView" name="listView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView {
    border: 1px hidden;
background-color: transparent;
	selection-background-color: rgba(0, 0, 0, 55);
}

QListView::item {
    width: 20px;
    height: 20px;
margin: 4px;
    subcontrol-position: center;
background-origin: padding;
background-clip : content;
border: 1px solid #191919;
padding: -1px;
}

QListView::item:!selected:hover {
border: 1px solid #191919;
}
QListView::item:selected {
border: 2px solid #191919;
margin: 1px;
}
QListView::item:hover {
border: 1px solid #191919;
margin: 2px;
}

QListView::icon {
height: 14px;
margin: -2px;
position: top;
subcontrol-position: top;
}
QListView::icon:selected {
    margin: 0px;
}
QListView::icon:hover {
    margin: 0px;
}
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QWidget" name="buttons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    margin: 0px;
padding: 0px;
min-width: 122px;
max-width: 122px;
min-height: 32px;
max-height: 32px;
border: 1px solid #191919;
border-radius: 3px;
font: 14px &quot;Open Sans Semibold&quot;;
color: #fafafa;
}
.QFrame {
    background-color: #455a64;
    border-radius: 4px;
border-bottom: 1px solid #445862;
}
EntityWidget {
    border: none;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="container_bttn_apply">
        <property name="minimumSize">
         <size>
          <width>124</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
          <widget class="QPushButton" name="bttn_apply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>124</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>124</width>
             <height>34</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #007828, stop:1 #00c853);
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(8, 140, 48, 255), stop:1 rgba(10, 222, 92, 255));
}
QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(18, 57, 31, 255), stop:1 rgba(43, 106, 69, 255));
}

QPushButton:disabled {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 120, 40, 100), stop:1 rgba(0, 200, 83, 100));
color: rgba(250, 250, 250, 100);
}

EntityWidget {
  background-color: rgba(255, 255, 255, 250);
}
EntityWidget:disabled {
  background-color: rgba(255, 255, 255, 100);
}</string>
           </property>
           <property name="text">
            <string>@bttn_apply</string>
           </property>
           <property name="shortcut">
            <string notr="true">Return</string>
           </property>
           <widget class="EntityWidget" name="solid_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>1</x>
              <y>1</y>
              <width>146</width>
              <height>1</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>146</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>146</width>
              <height>1</height>
             </size>
            </property>
            <property name="blending">
             <enum>E::Overlay</enum>
            </property>
            <property name="parentPainting">
             <enum>E::PaintParentBefore</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="container_bttn_cancel">
        <property name="minimumSize">
         <size>
          <width>124</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item alignment="Qt::AlignRight|Qt::AlignVCenter">
          <widget class="QPushButton" name="bttn_cancel">
           <property name="minimumSize">
            <size>
             <width>124</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>124</width>
             <height>34</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
  background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 255), stop:1 rgba(69, 89, 99, 255));
border: 1px solid #191919;
border-radius: 3px;
}

QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(205, 55, 33, 215), stop:0.33 rgba(69, 34, 32, 225), stop:1 rgba(69, 89, 99, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(41, 51, 54, 255), stop:0.5 rgba(18, 20, 26, 255), stop:1 rgba(24, 25, 32, 255));
}
QPushButton:disabled {
  background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 100), stop:1 rgba(69, 89, 99, 100));
color: rgba(250, 250, 250, 100);
}
EntityWidget {
  background-color: rgba(255, 255, 255, 250);
}
EntityWidget:disabled {
  background-color: rgba(255, 255, 255, 100);
}</string>
           </property>
           <property name="text">
            <string>@bttn_cancel</string>
           </property>
           <property name="shortcut">
            <string notr="true">Esc</string>
           </property>
           <widget class="EntityWidget" name="solid_8">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>1</x>
              <y>1</y>
              <width>146</width>
              <height>1</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>146</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>146</width>
              <height>1</height>
             </size>
            </property>
            <property name="blending">
             <enum>E::Overlay</enum>
            </property>
            <property name="parentPainting">
             <enum>E::PaintParentBefore</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MarkersListView</class>
   <extends>QListView</extends>
   <header>markerslist.h</header>
  </customwidget>
  <customwidget>
   <class>TextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>noteslisteditor.h</header>
   <slots>
    <signal>focused()</signal>
    <signal>unfocused()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>NotesListEditor</sender>
   <signal>focused()</signal>
   <receiver>textBrowser</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NotesListEditor</sender>
   <signal>focused()</signal>
   <receiver>textBrowser</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_apply</sender>
   <signal>clicked()</signal>
   <receiver>NotesListEditor</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>NotesListEditor</receiver>
   <slot>revert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>currentMarkerChanged(QModelIndex)</signal>
  <signal>editingStarted()</signal>
  <signal>editingFinished(bool)</signal>
  <slot>editNote(QModelIndex)</slot>
  <slot>submit()</slot>
  <slot>revert()</slot>
 </slots>
</ui>
